### DISCUSSION TITLE
How does V8 incorporate the dependency injection framework FX, in particular CTX = context.background

### DISCUSSION THREAD

**Discussion by peterwillcn** [redis/go-redis#2228] [Q&A]
How does V8 incorporate the dependency injection framework fx(go.uber.org/fx), in particular CTX = context.background？

**Comment by vmihailenco** [comment]
go-redis does not use fx. `context.Context` used for [observability](https://uptrace.dev/opentelemetry/go-tracing.html#context) purposes among others.
